src/content/docs/blog/roadmap-2025.md

```diff
@@ -0,0 +1,69 @@
+
+## 💳 Enterprise Support
+
+One more thing that we are happy to announce is that as of January 2025, we are also offering [Enterprise Support](https://biomejs.dev/enterprise) for Biome. Hopefully this will allow some of our contributors to spend more of their time and effort towards Biome!
+
+## ⏭️ Biome 2.0
+
+Right now our team is busy preparing for the Biome 2.0 release. Because our project is still run by volunteer contributors, we do not have an ETA for you. But we can share some of the goodies that will be coming:
+
+* **Plugins**. A long-requested feature, we started development of Biome plugins after an [RFC process](https://github.com/biomejs/biome/discussions/1762) that started January 2024. Biome 2.0 will feature the first fruits of this labor: Users will be able to create their own lint rules using [GritQL](https://docs.grit.io/language/overview).
+* **Domains**. [Domains](https://github.com/biomejs/biome/blob/next/.changeset/introduce_the_domains_linter_feature.md) are a configuration feature that makes it easy for users to enable or disable all rules related to a specific domain, such as React, Next.js or testing frameworks. It also allows Biome to automatically enable recommended domain-specific rules based on the dependencies listed in your `package.json`.
+* **Monorepo Support**. While support for monorepos was already improved with our `extends` feature in `biome.json`, many weak spots remained. Biome 2.0 has an improved architecture based on an internal `ProjectLayout` that should resolve most of these.
+* **Suppressions**. Biome already allowed *suppression* of linter diagnostics through the use of `// biome-ignore` suppression comments. With Biome 2.0 we're adding support for `// biome-ignore-all` and `// biome-ignore-start`/`biome-ignore-end` comments.
+* **Multi-file analysis**. Last but not least, we're adding true [Multi-file support](https://github.com/biomejs/biome/issues/3307) to Biome 2.0. This means that our lint rules will be able to query information from other files, which will enable much more powerful lint rules.
+
+## 🌌 2025 roadmap
+
+Again we should preface a disclaimer here: We're a community-driven project, so we cannot promise to deliver any of the features below. But that doesn't mean we don't have a wishlist of things we would like to work on in 2025 😉
+
+This year we will focus on:
+
+* [**HTML support**](https://github.com/biomejs/biome/issues/4726). No toolchain for the web is complete without it, and we're already working on it!
+* [**Language embedding**](https://github.com/biomejs/biome/issues/3334). CSS or GraphQL snippets inside a template literal in a JavaScript file? JavaScript or CSS inside an HTML file? Biome should be able to handle these as well, and we'll try to make it happen. This should also lead to better support for **Astro**, **Svelte**, and **Vue** than we have today.
```

Comment:
nit: the wording we've been using has been "Embedded languages", not "Language embedding". "Embedding" is [terminology overlap with machine learning](https://aws.amazon.com/what-is/embeddings-in-machine-learning/) and might be confusing